poi/poi_types.xml

```diff
@@ -819,7 +819,10 @@
 		</poi_category>
 		<poi_category name="finance" default_tag="finance">
 			<poi_type name="bank" tag="amenity" value="bank"/>
-			<poi_type name="atm" tag="amenity" value="atm"/>
+			<poi_filter name="atm">
```

Comment:
poi_filter used for predefined filters which are shown in search UI. poi_type - regular poi. poi_additional it "addition" to main tag. It is because our map data system.